src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -209,6 +229,54 @@ var HTMLDOMPropertyConfig = {
   },
   DOMPropertyNames: {
   },
+  DOMMutationMethods: {
+    value: function(node, next) {
```

Comment:
Ya. Took care of that in #7359. 

Hmm.. This PR is creating some extra noise for the input issue. Should I close this for now and resubmit after that settles?